modules/serverless-ossm-setup.adoc

```diff
@@ -17,45 +26,187 @@ metadata:
   name: default
   namespace: istio-system
 spec:
-  members:
-    - <namespace> <1>
+  members: <1>
+    - knative-serving
+    - <namespace>
 ----
 <1> A list of namespaces to be integrated with {ProductShortName}.
-. Create a network policy that permits traffic flow from Knative system Pods to Knative services:
-.. Add the `serving.knative.openshift.io/system-namespace=true` label to the `knative-serving` namespace:
++
+[IMPORTANT]
+====
+This list of namespaces must include the `knative-serving` namespace.
+====
+. Apply the `ServiceMeshMemberRoll` resource:
 +
 [source,terminal]
 ----
-$ oc label namespace knative-serving serving.knative.openshift.io/system-namespace=true
+$ oc apply -f <filename>
 ----
-.. Add the `serving.knative.openshift.io/system-namespace=true` label to the `knative-serving-ingress` namespace:
+. Create the necessary gateways so that {ProductShortName} accepts traffic:
++
+[source,yaml]
+----
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: knative-ingress-gateway
+  namespace: knative-serving
+spec:
+  selector:
+    istio: ingressgateway
+  servers:
+    - port:
+        number: 443
+        name: https
+        protocol: HTTPS
+      hosts:
+        - "*"
+      tls:
+        mode: SIMPLE
+        credentialName: <wildcard_certs> <1>
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+ name: knative-local-gateway
+ namespace: knative-serving
+spec:
+ selector:
+   istio: ingressgateway
+ servers:
+   - port:
+       number: 8081
+       name: http
+       protocol: HTTP <2>
+     hosts:
+       - "*"
+---
+apiVersion: v1
+kind: Service
+metadata:
+ name: knative-local-gateway
+ namespace: istio-system
+ labels:
+   experimental.istio.io/disable-gateway-port-translation: "true"
+spec:
+ type: ClusterIP
+ selector:
+   istio: ingressgateway
+ ports:
+   - name: http2
+     port: 80
+     targetPort: 8081
+----
+<1> Add the name of your wildcard certificate.
+<2> The `knative-local-gateway` serves HTTP traffic. That means that traffic coming from outside of {ProductShortName}, but using an internal hostname, such as `example.default.svc.cluster.local` is not encrypted. You can set up encryption for this path by creating another wildcard certificate and an additional gateway that uses a different `protocol` spec.
++
+.Example `knative-local-gateway` using HTTPS
+[source,yaml]
+----
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: knative-local-gateway
+  namespace: knative-serving
+spec:
+  selector:
+    istio: ingressgateway
+  servers:
+    - port:
+        number: 443
+        name: https
+        protocol: HTTPS
```

Comment:
Need a call-out here to emphasize that it is different from the previous example?

Reply:
I'll follow this up another time if that's OK - I think for now it's enough to call out in the headings that they're different